about_first_project.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" type="text/CSS" href="about_first_project.css"/> 
+    <link href="https://fonts.googleapis.com/css2?family=Nunito:wght@300&display=swap" rel="stylesheet">
+</head>
+<body>
+    
+<header>  
+    <h1> About Moi....</h1>  
+</header>
+<div class="pink_box"> 
```

Comment:
A suggestion is to change class to `pink-box` (using kebab-case for CSS element)